packages/mui-system/src/spacing.js

```diff
@@ -93,7 +93,11 @@ const paddingKeys = [
 const spacingKeys = [...marginKeys, ...paddingKeys];
 
 export function createUnaryUnit(theme, themeKey, defaultValue, propName) {
-  const themeSpacing = getPath(theme, themeKey) ?? defaultValue;
+  let themeSpacing = getPath(theme, themeKey) ?? defaultValue;
+  if (typeof themeSpacing === 'string') {
+    // Check the non CSS var value
+    themeSpacing = getPath(theme, themeKey, false) ?? defaultValue;
+  }
```

Comment:
I think this can be shorten to:

```suggestion
  const themeSpacing = getPath(theme, themeKey, !theme.vars) ?? defaultValue;
```

because `createUnaryUnit` does not work with CSS variables yet.

Reply:
Makes sense 👍